zigate/cluster/core.py

```diff
@@ -37,6 +38,7 @@ def __init__(self, device, endpoint):
         self._attributes = self.__class__._attributes.copy()
         self.device = device
         self.endpoint = endpoint
+        self.attr_notify = signal('zigate_attr_notify')
```

Comment:
Perhaps we can move the signals to their own module, like `zigate/signals.py` and create one global signal for each event. Then when dispatching the event, add the `Cluster` instance as keyword arg.

And maybe you can give it a more descriptive name like `attribute_changed`.